scripts/commit-api-spec-change-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+echo "Running update changelog script"
+changeLog="$1"
+echo "$changeLog"
+node scripts/update-change-log.js "$changeLog"
+echo "Git configurations"
+git config --global user.email "team_interfaces+github@twilio.com"
```

Comment:
[Q] What do you think about keeping the user.email and user.name in environment variables?

Reply:
Yeah, We can do that. We don't need to make a code change if we want to change name/email.